src/modules/CreateImage/components/CreateImageInitializationHeaderComponent/CreateImageInitializationHeaderComponent.tsx

```diff
@@ -7,36 +7,108 @@ import Flex from 'common/components/Flex/Flex';
 
 import styles from './createImageInitializationHeaderComponent.scss';
 
+import capitalize from 'utils/capitalize';
```

Comment:
reorder, please, according to this guide https://trello.com/c/VcHAzRFZ/6-i-want-us-to-have-a-certain-import-files-order